app/src/main/java/com/example/taskyy/ui/viewmodels/LoginViewModel.kt

```diff
@@ -90,14 +90,14 @@ class LoginViewModel @Inject constructor(
 }
 
 data class LoginState(
-    var email: String = "",
-    var password: String = "",
-    var name: String = "",
-    var isLogginIn: Boolean = false,
-    var isEmailValid: Boolean = false,
-    var isPasswordVisible: Boolean = false,
-    var isLoginSuccessful: Boolean = false,
-    var loginErrorMessage: UiText? = null,
-    var isReady: Boolean = false
+    val email: String? = null,
+    val password: String? = null,
```

Comment:
When text field values are empty it's very clear that it refers to an empty text field, since it's a valid state for a text field value (unlike an empty string for an error message which is not a valid error). So here, the default values can stay empty strings. Null values would rather give me the feel that the whole text field wouldn't exist in that case